packages/aws-cdk/README.md

```diff
@@ -433,6 +433,18 @@ and might have breaking changes in the future.
 
 **⚠ Note #3**: Expected defaults for certain parameters may be different with the hotswap parameter. For example, an ECS service's minimum healthy percentage will currently be set to 0. Please review the source accordingly if this occurs.
 
+**⚠ Note #4**: Only usage of certain [CloudFormation intrinsic functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html) are supported as part of a hotswapped deployment. At time of writing, these are:
+
+- `Ref`
+- `Fn::GetAtt` *
+- `Fn::ImportValue`
+- `Fn::Join`
+- `Fn::Select`
+- `Fn::Split`
+- `Fn::Sub`
+
+> *: `Fn::GetAtt` is only partially supported.
```

Comment:
If possible (Not required for this PR), can you mention what the partial support looks like?

Reply:
Hey @vinayak-kukreja the best clarification would be a link to [the code](https://github.com/aws/aws-cdk/blob/1929f2e83d14ddcd6123bb9b522456a8c1bd5772/packages/aws-cdk/lib/api/evaluate-cloudformation-template.ts#L477) that implements it but that may not be suitable for this user-facing doco. I could list partially supported resources but that has a chance of becoming stale doco. Thoughts?

cc @tmokmss

Reply:
Ok, so there can be a link here. For instance, saying `see here[the link embedded] for more information`?

Reply:
Sure thing, can do!